packages/jaeger-ui/src/components/TraceDiff/TraceDiffHeader/TraceTimelineLink.test.js

```diff
@@ -13,30 +13,31 @@
 // limitations under the License.
 
 import * as React from 'react';
-import { shallow } from 'enzyme';
+import { render, screen, fireEvent, createEvent } from '@testing-library/react';
```

Comment:
```suggestion
import { render, screen, fireEvent, createEvent } from '@testing-library/react';
import '@testing-library/jest-dom';
```

Reply:
Why do we import the `jest-dom` environment here?

Reply:
Functions like `toHaveAttribute` can be used with that import

Reply:
ah, with this ^ the other suggestion works. But I prefer a more explicit / brute-force `new URL().pathname`.